test-case/composer.json

```diff
@@ -17,7 +17,6 @@
     },
     "require-dev": {
         "ext-pdo": "*",
-        "contao/core-bundle": "self.version",
```

Comment:
This reverts https://github.com/contao/contao/pull/7751 - intended?

Reply:
No, it doesn't. The requirement has been moved from `require-dev` to `require`.